src/Pages/Review/components/TicketCard/TicketCard.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { addToBasket } from '../../actions';
+import './ticketcard.scss';
+
+export class TicketCardComponent extends Component {
+    constructor (props) {
+        super(props);
+        this.state = {};
+    }
+
+    handleClick = (item) => {
+        this.props.dispatch(addToBasket(item));
+        this.setState({
+            active: true,
+        });
+    }
+
+    render () {
+        const { product: { id, title, price } } = this.props;
+        const activeClass = this.state.active ? 'card active' : 'card';
+        return (
+            <div
+                className={activeClass}
+                onClick={() => this.handleClick({id, title, price})}
```

Comment:
creating a function in the render method is an performance anti pattern. if you need to pass variables to the handler you can use data attributes:
```
constructor() {
  ...
  this.handleClick = this.handleClick.bind(this);
}
handleClick(event) {
  const { id, title, price } = event.target.dataset;
  const item = {
    id,
    title,
    price,
  };
  this.props.dispatch(addToBasket(item));
  this.setState({
    active: true,
  });
}
render () {
  ...
  <div
    className={activeClass}
    data-id={id}
    data-title={title}
    data-price={price}
    onClick={this.handleClick}
  >
}